bin/sirius-script-mamba-env-create.bash

```diff
@@ -0,0 +1,580 @@
+$COMM pyparsing bottleneck aiohttp==3.7.4 scipy matplotlib pytest mpmath \
+    entrypoints requests pyvisa=1.10.1 pyvisa-py=0.3.1 pyqt=5.12.3 pandas \
+    pyqtgraph=0.11.0 QtAwesome=0.7.2 numexpr tk sh pywavelets scikit-image \
+    scikit-learn pydocstyle pycodestyle pylama
+
+
+printf_yellow_clear "- Install EPICS Base:\n"
+$COMM -c conda-forge/label/cf202003 epics-base=3.15.6
+
+printf_yellow_clear "- And some other EPICS packages:\n"
+$COMM pyepics=3.5.0 pcaspy==0.7.3 pydm=1.10.3 timechart=1.2.3
+# remove the activate and deactivate files created by pyepics and pydm:
+cd $CONDA_PREFIX/etc/conda/activate.d
+if [ -f "pydm.bat" ]
+then
+    rm pydm.bat pydm.sh pyepics_activate.sh epics_base.sh
+fi
+cd $CONDA_PREFIX/etc/conda/deactivate.d
+if [ -f "pydm.sh" ]
+then
+    rm pydm.sh pyepics_deactivate.sh epics_base.sh
+fi
+
+printf_yellow_clear "- Install and configure jupyter notebook\n"
+$COMM jupyter notebook
+mamba update jupyter_client
+$COMM jupyter_contrib_nbextensions
+
+### Clone and install our repositories
+if [ "$CLONE" == "yes" ]
+then
+    printf_yellow "Clone and install our applications.\n"
+    printf_yellow_clear "Creating folder $CONDA_PREFIX/repos: "
+    if ! [ -d $CONDA_PREFIX/repos ]
+    then
+        mkdir $CONDA_PREFIX/repos
+        printf_green "done!\n"
+    else
+        printf_blue "already exists. Skipping...\n"
+    fi
+else
+    printf_yellow "Find and install our applications.\n"
+fi
+
+TARGET="install"
+printf_yellow "\nSIRIUS packages will be installed in: "
+if [ "$DEVELOP" == "yes" ]
+then
+    TARGET="develop-install"
+    printf_green "Develop mode.\n\n"
+else
+    printf_green "Install mode.\n\n"
+fi
+
+printf_yellow_clear "Installing SIRIUS Control System related packages.\n"
+clone_or_find mathphys lnls-fac && make $TARGET
+clone_or_find dev-packages lnls-sirius && cd siriuspy && \
+    make $TARGET
+clone_or_find hla lnls-sirius && cd pyqt-apps && make $TARGET
+clone_or_find hlafac lnls-fac && make $TARGET
+if [ "$INST_SIM" == "yes" ]
+then
+    printf_yellow_clear "Installing accelerators simulation packages.\n"
+    clone_or_find lnls lnls-fac && make $TARGET
+    clone_or_find trackcpp lnls-fac && make clean && \
+        make install-cpp 2>/dev/null && make $TARGET-py 2>/dev/null
+    clone_or_find pyaccel lnls-fac && make $TARGET
+    clone_or_find pymodels lnls-fac && make $TARGET
+    clone_or_find apsuite lnls-fac && make $TARGET
+fi
+if [ "$INST_COL" == "yes" ]
+then
+    printf_yellow_clear "Installing collective effects simulation packages.\n"
+    clone_or_find collective_effects lnls-fac && cd cppcolleff && \
+        make clean && make install-cpp 2>/dev/null && \
+        make $TARGET-py 2>/dev/null && \
+        cd ../pycolleff && make $TARGET
+fi
+if [ "$INST_IOC" == "yes" ]
+then
+    printf_yellow_clear "Installing SIRIUS IOCs related packages.\n"
+    clone_or_find eth-bridge-pru-serial485 lnls-sirius && cd client && \
+        pip install --no-deps -e ./
```

Comment:
I think we could use the same  commands (make $TARGET) for eth-bridge, this command was used before because its Makefile was not following the standard